manifests/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -267,6 +267,36 @@ data:
     networks: {}
   {{- end }}
 
+  Corefile: |-
+
+    global:15054 {
```

Comment:
This needs to be configurable.

Reply:
You mean the "global" constant ? Yes, as noted in another comment the ingress is hardcoding it as well - I think a separate PR that adds something to mesh config would be best, taking into account what K8S multicluster is planning as well. 

However - this setting is no longer needed. I was initially thinking that the coreDNS sidecar would terminate DNS requests and forward .global to istiod - but it's too complicated to set the TLS and looking a bit at the coredns implementation I'm no longer considering this path. So I removed .global from here - there requests start in agent, go to istiod ( and I'm planning to evaluate sending them over DNS-HTTP or even XDS ), and .global and clusters will be resolved by istiod from it's own database. But I don't know how long it'll take - if I can finish
in 1.6 I can just remove the coredns sidecar.